packages/diagrams/lib/src/blend_mode.dart

```diff
@@ -16,9 +16,9 @@ import 'diagram_step.dart';
 Completer<Null> touch;
 final GlobalKey key = new GlobalKey();
 
-const String destinationImageName = 'assets/blend_mode_destination.jpeg';
-const String sourceImageName = 'assets/blend_mode_source.png';
-const String gridImageName = 'assets/blend_mode_grid.png';
+const ImageProvider destinationImageProvider = const ExactAssetImage('assets/blend_mode_destination.jpeg', package: 'diagrams');
```

Comment:
I don't think you need the second const here.

Reply:
oops.